rwslib/builders/admindata.py

```diff
@@ -0,0 +1,391 @@
+# -*- coding: utf-8 -*-
+
+from rwslib.builders.common import ODMElement, dt_to_iso8601
+from rwslib.builders.clinicaldata import LocationRef
+from rwslib.builders.constants import LocationType, UserType
+from rwslib.builders.modm import LastUpdateMixin
+
+
+class AdminData(ODMElement):
+    """
+    Administrative information about users, locations, and electronic signatures.
+    """
+    def __init__(self, study_oid=None):
+        """
+        :param str study_oid: OID pointing to the StudyDef
+        """
+        super(AdminData, self).__init__()
+        self.study_oid = study_oid
+        self.users = []
+        self.locations = []
+        # SignatureDef
+
+    def build(self, builder):
+        """Build XML by appending to builder"""
+        params = {}
+        if self.study_oid:
+            params.update(dict(StudyOID=self.study_oid))
+        builder.start("AdminData", params)
+        for user in self.users:
+            user.build(builder)
+        for location in self.locations:
+            location.build(builder)
+
+        builder.end("AdminData")
+
+    def __lshift__(self, other):
```

Comment:
/ian floats in from beyond the grave...

I agree this is "unpythonic" but at the same time when I came up with this idiom (if indeed I didn't steal it from something else, xist maybe?) the goal was to have an operator which means "push X into Y as a child". The syntax "x << y" seemed like a succinct way to do that that could be chained in a natural way. If you take a look at the builders docs:

http://rwslib.readthedocs.io/en/latest/using_builders.html

There's an example:

```python
odm << clinical_data << subject_data << event_data << form_data << itemgroup
```
Which to me is clear and better than an "update" alternative:

```python
odm.update(clinical_data.update(subject_data.update(event_data.update(form_data.update(itemgroup)))))
```
It was just a bit too lispy for my liking. But you can do it that way with the overloaded __call__ if you want to. And in the example I mix that in the styles too:
```python
meta << ItemDef("SEX", "Gender", DATATYPE_TEXT, 1, control_type=ItemDef.CONTROLTYPE_RADIOBUTTON
   )(
    Question()(TranslatedText("Gender at Birth")),
    CodeListRef("CL_SEX")
)
```
Which is ugly as sin I admit, which is why I prefer(red) the <<.  

But also, update() is a dictionary thing that would lead you to believe that there's some kind of merge of values but this isn't the same thing, it's a .children.append() operation.

Great catches elsewhere in this review regarding default constructor parameters!